train.ipynb

```diff
@@ -131,7 +131,7 @@
     "    \n",
     "    model.train()\n",
     "\n",
-    "    dec_log_probs, dec_hidden = model(input, target, inputs_len, target_len)\n",
+    "    dec_log_probs, dec_hidden, best_attn_labels = model(input, target, inputs_len, target_len)\n",
```

Comment:
as i said above, let's change this so the model returns attention scores instead and we implement post processing in the train loop. IMO for cleaner code and better debugging you can create `inference.py` and put a function that does post processing in there.